src/clients/openapi.js

```diff
@@ -66,11 +66,13 @@ export function buildHeaders(context, req) {
     return headers;
 }
 
-function validateResponse(response) {
+export function validateResponse(response) {
     const contentType = get(response, 'headers["content-type"]');
     if (contentType !== 'application/json') {
-        throw new Error(`${contentType} is not a valid response content-type`);
+        const message = `${contentType} is not a valid response content-type`;
```

Comment:
Not sure why you need two variables here, looks like the message is enough.
You can send back the error message if there was an error , if there wasn't then return an empty message (null).

Reply:
I'm thinking about the next errors - that are not response type. Maybe it's too much - but...